### PR TITLE
More translations from undead dragon, extracted ui element translations

### PR DESCRIPTION
**Pull request by Vortilion**

### CODE REVIEW

File: src/components/CampaignExport.vue [hunk 1]
```diff
@@ -52,12 +54,12 @@ function closeModal() {
...
     @click="openModal"
   >
-    Export campaign
+    {{ t("label.export-campaign") }}
   </button>
   <BaseModal :is-open="isOpen" @close-modal="closeModal">
     <template #header>
       <div class="grid grid-cols-2">
-        <div class="w-full font-medium place-self-center">Export campaign</div>
+        <div class="w-full font-medium place-self-center">{{}}</div>
```
Comment (marcojanssen) [inline src/components/CampaignExport.vue]: Translation is missing

File: src/components/CampaignImport.vue [hunk 2]
```diff
@@ -78,12 +80,12 @@ function closeModal() {
...
     @click="openModal"
   >
-    Import campaign
+    {{ t("label.import-campaign") }}
   </button>
   <BaseModal :is-open="isOpen" @close-modal="closeModal" id="campaign-import-modal">
     <template #header>
       <div class="grid grid-cols-2">
-        <div class="w-full font-medium place-self-center">Import campaign</div>
+        <div class="w-full font-medium place-self-center">{{}}</div>
```
Comment (marcojanssen) [inline src/components/CampaignImport.vue]: Translation is missing

File: src/components/CampaignNew.vue [hunk 3]
```diff
@@ -37,12 +39,12 @@ function newCampaign(campaign: "core" | "apocalypse" | "awakenings") {
...
     @click="openModal"
   >
-    New campaign
+    {{ t("label.new-campaign") }}
   </button>
   <BaseModal :is-open="isOpen" @close-modal="closeModal">
     <template #header>
       <div class="grid grid-cols-2">
-        <div class="w-full font-medium place-self-center">New campaign</div>
+        <div class="w-full font-medium place-self-center">{{}}</div>
```
Comment (marcojanssen) [inline src/components/CampaignNew.vue]: Translation is missing

File: src/components/CampaignRemove.vue [hunk 4]
```diff
@@ -38,12 +40,12 @@ function removeCampaign() {
...
     @click="openModal"
   >
-    Remove campaign
+    {{ t("label.remove-campaign") }}
   </button>
   <BaseModal :is-open="isOpen" @close-modal="closeModal">
     <template #header>
       <div class="grid grid-cols-2">
-        <div class="w-full font-medium place-self-center">Remove campaign</div>
+        <div class="w-full font-medium place-self-center">{{}}</div>
```
Comment (marcojanssen) [inline src/components/CampaignRemove.vue]: Translation is missing

**Review by marcojanssen — CHANGES_REQUESTED** [review]

File: src/components/CampaignExport.vue (hunk 1 above)
Comment (Vortilion) [reply src/components/CampaignExport.vue]: Done.

File: src/components/CampaignImport.vue (hunk 2 above)
Comment (Vortilion) [reply src/components/CampaignImport.vue]: done.

File: src/components/CampaignNew.vue (hunk 3 above)
Comment (Vortilion) [reply src/components/CampaignNew.vue]: done.

File: src/components/CampaignRemove.vue (hunk 4 above)
Comment (Vortilion) [reply src/components/CampaignRemove.vue]: done.

**Review by marcojanssen — APPROVED** [review]

**Pull request merged**